### DISCUSSION TITLE
Do not recommend starred repos

### DISCUSSION THREAD

**Discussion by artalar** [community/community#15822] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! Feed is interesting feature, thx for it. But I've just got recommendation of already starred repo, it is kinda useless :)

<img width="722" alt="image" src="https://user-images.githubusercontent.com/27290320/166070423-02419006-f93d-4390-8393-2f4cfd32b729.png">

**Comment by ghost** [comment]
Fully agreed. There is also an issue in which Repos starred by those you follow will also appear on both "following" and "for you" sections which may be unintuitive.

**Comment by jfullstackdev** [comment, marked as answer]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by jfullstackdev** [comment]
as for the starred repos that appear on Recommendations, it's a known issue and the Staff is already working on it
